musicTheory.ts

```diff
@@ -52,6 +56,7 @@ const noteValues: Record<Note, number> = {
   B: 11,
 };
 
+// REVIEW: CONSTANT_CASE preferred here
```

Comment:
I’ll change these (including above) to be constant case. That’s for the name of the function right? So accidentalValues should be ACCIDENTAL_VALUES ?